src/System.ValueTuple/tests/ValueTuple/UnitTests.cs

```diff
@@ -718,9 +718,12 @@ public static void ZeroTuples()
     {
         var a = ValueTuple.Create();
         Assert.True(a.Equals(new ValueTuple()));
+        Assert.Equal(0, a.CompareTo(new ValueTuple()));
 
         Assert.Equal(1, ((IStructuralComparable)a).CompareTo(null, DummyTestComparer.Instance));
         Assert.Throws<ArgumentException>(() => ((IStructuralComparable)a).CompareTo("string", DummyTestComparer.Instance));
+
+        Assert.Equal("(1, 2, 3, 4, 5, 6, 7, )", CreateLong(1, 2, 3, 4, 5, 6, 7, new ValueTuple()).ToString());
```

Comment:
Another option would be to print `(1, ..., 7, ())`. 
The trouble with that is it becomes inconsistent. If you have rest = VT0, then you get extra parentheses, and if you have rest = VT1 or VT2, the parentheses disappear.

@gafter @AlekseyTs @VSadov This relates to our discussion on 0-tuples.